### PR TITLE
Gh action unit test coverage

### PR DESCRIPTION
**Pull request by vishwajeetraj11**

### CODE REVIEW

**Review by paul4702 — APPROVED** [review]

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -7,16 +7,37 @@ jobs:
...
-          VITE_TEST_CP_URL_2: ${{ secrets.VITE_TEST_CP_URL_2 }}
+    - uses: actions/checkout@v4
+    - uses: ./.github/actions/ci-setup
+    - name: 'Test'
+      run: npm run test:unit
+      env:
+        VITE_PROJECT_ID: ${{ secrets.VITE_PROJECT_ID }}
+        VITE_SEED_PHRASE: ${{ secrets.VITE_SEED_PHRASE }}
+        VITE_WALLET_PASSWORD: ${{ secrets.VITE_WALLET_PASSWORD }}
+        VITE_TEST_CP_URL_1: ${{ secrets.VITE_TEST_CP_URL_1 }}
```
Comment (souvikmishra) [inline .github/workflows/test.yml:27]: are CP url and seed phrases required for unit tests?

File: package.json [hunk 2]
```diff
@@ -76,6 +77,7 @@
 		"crypto-browserify": "^3.12.0",
 		"daisyui": "^3.6.4",
 		"ethers": "^6.12.1",
+		"json-summary": "^1.3.0",
```
Comment (souvikmishra) [inline package.json:80]: could this be a dev dependancy?

Comment (vishwajeetraj11) [reply package.json]: removed, since its not required.

File: .github/workflows/test.yml (hunk 1 above)
Comment (vishwajeetraj11) [reply .github/workflows/test.yml]: resolved.

**Review by souvikmishra — APPROVED** [review]

**Pull request merged**